qemu/lib/cdrom.py

```diff
@@ -0,0 +1,108 @@
+"""
+cdrom event check utility functions.
```

Comment:
for cdrom.py, it should cover all cdrom relevant functions.
so I think the description above is better to add to function.